.gitignore

```diff
@@ -1,3 +1,4 @@
 htmldoc
+katex
```

Comment:
The intention was to avoid recursive file listings by `git status` when `katex.tar.gz` has been downloaded und unpacked to get SsKaTeX covered in the tests. Turns out that `git status` lists only `katex` if that line is missing, no matter how big the tree below that is (in contrast to e.g. monotone's `mtn list unknown`), so I'll remove it.